test/allocs.jl

```diff
@@ -30,7 +30,7 @@ end
 
 if Sys.isunix()
   @testset "Allocation tests" begin
-    @testset "$symsolver" for symsolver in (:LBFGSSolver, :R2Solver, :TrunkSolver, :TronSolver)
+    @testset "$symsolver" for symsolver in (:LBFGSSolver, :FoSolver, :FomoSolver, :TrunkSolver, :TronSolver)
```

Comment:
if you use
```
if symsolver == :FomoSolver
  T = eltype(nlp.meta.x0)
  stats = GenericExecutionStats(nlp, solver_specific = Dict(:avgβmax => T(0)))
else
  stats = GenericExecutionStats(nlp)
end
```
on L.38 instead of `stats = GenericExecutionStats(nlp)`, the allocation test should pass.